.github/workflows/maven.yml

```diff
@@ -29,13 +29,23 @@ jobs:
         restore-keys: ${{ runner.os }}-m2
     - name: Test with Maven
       run: mvn --batch-mode --update-snapshots test
-      
+
   build:
     runs-on: ubuntu-latest
 
     if: "contains(github.event.head_commit.message, '[Deploy]')"
 
     steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 11
+      uses: actions/setup-java@v2
+      with:
+        java-version: '11'
```

Comment:
Zmiany w tym pliku miały zostać zignorowane, teraz to czyste marnowanie czasu na CI